### DISCUSSION TITLE
Google Sheets Image (Graphs) Support

### DISCUSSION THREAD

**Discussion by tplowe56** [archetyped/simple-lightbox-extensions#120] [Extension Requests]
Extension Request

Google sheets URL's provided for publishing a chart/image is not in a standard image format, so SLB does not detect it as an image to be displayed in a lightbox. 

The chart/image is displayed properly but has no navigation controls, so the user has to use the back button on pc/mobile to exit the image.

Example: https://www.loweprofile.com/investing101/performance/

**Comment by archetyped** [comment]
Example image URL format:
```
https://docs.google.com/spreadsheets/d/e/2PACX-1vRVxTY9bgTFEAf6moBJb-BKHATbN3ySN4KfV9YY09dvaC3_8aOHqcOZTQP3_CmUSGvFtc4YPTK8YvYo/pubchart?oid=903272961&format=image
```
